modules/ui/src/main/scala/lucuma/ui/primereact/CheckboxView.scala

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.ui.primereact
+
+import cats.syntax.all._
+import crystal.react.View
+import eu.timepit.refined.types.string.NonEmptyString
+import japgolly.scalajs.react.*
+import japgolly.scalajs.react.vdom.html_<^._
+import react.common.*
+import react.primereact.Checkbox
+
+import scalajs.js
+
+case class CheckboxView(
+  id:       NonEmptyString,
+  value:    View[Boolean],
+  label:    String,
+  inputId:  js.UndefOr[String] = js.undefined, // id of the input element
+  disabled: js.UndefOr[Boolean] = js.undefined,
+  clazz:    js.UndefOr[Css] = js.undefined
+) extends ReactFnProps[CheckboxView](CheckboxView.component)
```

Comment:
Just FYI: the type parameter in no longer necessary in `React(Fn)Props`, Scala 3 now infers it.

Reply:
Oh, nice.